README.md

```diff
@@ -85,6 +85,9 @@ You can check the development status at:
   - [HTTP-API](#http-api)
   - [IPFS Core (use IPFS as a module in Node.js or in the Browser)](#ipfs-core-examples-use-ipfs-as-a-module)
     - [Create a IPFS node instance](#create-a-ipfs-node-instance)
+    - [Add a file](#add-a-file)
+    - [Retrieve a file](#retrieve-a-file)
```

Comment:
I believe this is well demonstrated in the examples folder, we can trim the README by just pointing to those examples.